release-team/role-handbooks/communications/README.md

```diff
@@ -39,7 +39,7 @@ The following groups should be members:
 * The current release cycle's Communications Lead & Comms Shadows
 * SIG Release Chairs
 
-The list hast to be rotated/actively managed every cycle.
+The list has to be rotated/actively managed every cycle. Submit a PR to update <a href="https://git.k8s.io/k8s.io/groups/groups.yaml">this document</a> per the milestone activities described below.
```

Comment:
I suggest using markdown links instead of HTML:
```suggestion
The list has to be rotated/actively managed every cycle. Submit a PR to update [this document]("https://git.k8s.io/k8s.io/groups/groups.yaml) per the milestone activities described below.
```

Reply:
@saschagrunert : This has been changed across the handbook for Comms. PTAL & let me know if it looks okay.

Reply:
This one is fine, thank you! The one below needs a fix :point_down: